build.gradle

```diff
@@ -186,21 +186,21 @@ android {
 
         release {
             resValue "string", "app_name_buildtype", "@string/app_name"
-            resValue "string", "authority", "org.owncloud"
-            resValue "string", "search_suggest_authority", "org.owncloud.search.users_and_groups"
-            resValue "string", "document_provider_authority", "org.owncloud.documents"
-            resValue "string", "file_provider_authority", "org.owncloud.files"
+            resValue "string", "dynamic_authority", "@string/authority"
+            resValue "string", "dynamic_search_suggest_authority", "@string/search_suggest_authority"
+            resValue "string", "dynamic_document_provider_authority", "@string/document_provider_authority"
+            resValue "string", "dynamic_file_provider_authority", "@string/file_provider_authority"
             if (System.env.OC_RELEASE_KEYSTORE) {
                 signingConfig signingConfigs.release
             }
         }
 
         debug {
             resValue "string", "app_name_buildtype", "Debug Owncloud"
-            resValue "string", "authority", "org.owncloud.debug"
-            resValue "string", "search_suggest_authority", "org.owncloud.search.users_and_groups.debug"
-            resValue "string", "document_provider_authority", "org.owncloud.documents.debug"
-            resValue "string", "file_provider_authority", "org.owncloud.files.debug"
+            resValue "string", "dynamic_authority", "org.owncloud.debug"
```

Comment:
But here we need something like `"@string/search_suggest_authority" + ".debug"` instead of hardcoded `"org.owncloud.search.users_and_groups.debug"`

Reply:
Following your suggestion 
<img width="740" alt="image" src="https://user-images.githubusercontent.com/3314607/50000059-aac5c600-ff98-11e8-8401-9badcdefc04f.png">
it looks like 
<img width="882" alt="image" src="https://user-images.githubusercontent.com/3314607/50000070-b74a1e80-ff98-11e8-8dc7-39bc052413f8.png">

So it doesn't work and I found no beautiful solution. So I keep it in current state.
If you know how to do it elegant, you can add a commit or make a follow up PR

Reply:
@davigonz ?

Reply:
Just noticed that `@string/whatever` without .debug suffix is not working either, I've tried to put `@string/app_name` within debug buildType and this is what I see in generated.xml file.

![screenshot 2018-12-19 at 10 00 05](https://user-images.githubusercontent.com/15261363/50209796-dff55e00-0374-11e9-97c9-3ba21a486f00.png)

Reply:
That's ok !
It a placeholder which used later. It's like to use @string/something in a xml. 
Please make a test, if you don't trust